cross-chain/arbitrum/external/mainnetEthereum/TokenBridge.json

```diff
@@ -0,0 +1,74 @@
+{
+  "address": "0x3ee18B2214AFF97000D974cf647E7C347E8fa585",
```

Comment:
Double-checked with https://book.wormhole.com/reference/contracts.html#token-bridge and confirmed the address is `0x3ee18B2214AFF97000D974cf647E7C347E8fa585`.